earth_enterprise/src/common/khCache.h

```diff
@@ -205,7 +205,7 @@ class khCache {
   }
   // calculates the current size of a cache item
   uint64 calculateCacheItemSize(Item *item) {
-    return khCacheItemSize + GetObjectSize(item->key) + GetObjectSize(item->val);
+    return khCacheItemSize + GetHeapUsage(item->key) + GetHeapUsage(item->val);
```

Comment:
On a different subject, I was thinking about your concern about the performance cost of always calculating memory usage. We can get around that pretty easily by checking `limitCacheMemory` in this function. If it's true, we calculate sizes like we're doing now, and if it's false, we return 0. That's a simple enough fix that I think it's worth doing.

Reply:
Might be a good idea to not include that info in getop as well or else it will report no memory used if I'm not mistaken and that would be confusing.

Reply:
Good point.